usaspending_api/etl/es_transaction_mapping.json

```diff
@@ -0,0 +1,270 @@
+{
+  "settings": {
+    "index.mapping.ignore_malformed": true,
+    "index.max_result_window": 50000,
+    "index.refresh_interval": -1,
+    "index": {
+      "number_of_shards": 5,
+      "number_of_replicas": 0
```

Comment:
Having 0 replicas is a bad idea

Reply:
It's a faster ingest, this is in the event recreate is true and index doesn't already exist.